internal/pkg/service/buffer/store/key/key.go

```diff
@@ -48,7 +48,7 @@ type RecordKey struct {
 type TaskKey struct {
 	ReceiverKey
 	Type   string `json:"type" validate:"required"`
-	TaskID TaskID `json:"taskId" validate:"required"`
+	TaskID TaskID `json:"taskId"` // validate:"required"
```

Comment:
Tasky aktualne nacitavame iba pre `cleanup`, inak sluzia iba ako nejaky log.
(To sa zmeni tou tvojou pracou a create reciver/export).

Preto musime docasne vypnut tuto validaciu, pretoze stare tasky tento field nemaju.

Ked to (v pondelok) nasadime, tak sa zacne spustat cleanup a stare tasky sa vymazu, potom to vratime spat.

Je to preto, lebo objekty validujeme aj pri nacitani z etcd.